Protos/V1/sessions_service.proto

```diff
@@ -0,0 +1,96 @@
+syntax = "proto3";
+
+import "objects.proto";
+import "session_status.proto";  
+
+option csharp_namespace = "ArmoniK.Api.gRPC.V1.Sessions";
+package ArmoniK.api.grpc.V1.sessions;
+
+/**
+ * The SessionService provides methods for listing, getting and cancelling sessions
+ */
+service Sessions {
+  // Get all sessions with pagination
+  rpc ListSessions(ListSessionsRequest) returns (ListSessionsResponse);
+
+  // Get a session using an ID
+  rpc GetSession(GetSessionRequest) returns (GetSessionResponse) {}
+
+  // Cancel a session using an ID
+  rpc CancelSession(CancelSessionRequest) returns (CancelSessionResponse) {}
+}
+
+/**
+ * A raw session object
+ */
+message RawSession {
+  // à compléter avec le schéma de la base de données
+}
+
+/**
+ * A session object
+ */
+message SessionTable {
+  // à compléter avec le schéma de la base de données
+}
+
+/**
+ * Request for listing sessions
+ */
+message ListSessionsRequest {
+  // Page number
+  string page_token = 1;
+  // Page size
+  int32 page_size = 2;
+  
+  // Name of the application
```

Comment:
Add a filter message for the following fields? (except order)
Just to have a nice scoping of the fields.